packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-validation.spec.ts

```diff
@@ -114,258 +113,266 @@ describe('DbasS API Error Handling', () => {
-        cy.intercept('GET', apiMatcher(`/monitor/dashboards/${id}`), {
-          statusCode: statusCode,
-          body: {
-            errors: [
-              {
-                reason: 'Bad Request',
-              },
-            ],
-          },
-        }).as('fetchDashboardById');
-
-        cy.visitWithLogin('monitor/cloudpulse');
-
-        //  Select a dashboard from the autocomplete input. Verify that the input is visible before typing.
-        ui.autocomplete
-          .findByLabel('Dashboard')
-          .should('be.visible')
-          .type(`${dashboardName}{enter}`)
-          .should('be.visible');
-
-        //  Select a database engine from the autocomplete input. Verify visibility before interaction.
-        ui.autocomplete
-          .findByLabel('Database Engine')
-          .should('be.visible')
-          .type(`${engine}{enter}`)
-          .should('be.visible');
-
-        //  Select a region from the dropdown. Verify visibility before interaction.
-        ui.regionSelect.find().click().type(`${region}{enter}`);
-
-        // Select a database cluster from the autocomplete input. Verify visibility before interaction.
-        ui.autocomplete
-          .findByLabel('Database Clusters')
-          .should('be.visible')
-          .type(`${clusterName}{enter}`)
-          .click();
-        cy.findByText(clusterName).should('be.visible');
-
-        //  Select a node type from the autocomplete input. Verify visibility before interaction.
-        ui.autocomplete
-          .findByLabel('Node Type')
-          .should('be.visible')
-          .type(`${nodeType}{enter}`);
-
-        //  Wait for the API calls to fetch services and dashboard to resolve.
-        cy.wait(['@fetchServices', '@fetchDashboard']);
-
-        cy.get('[data-qa-error-msg="true"]')
-          .should('be.visible')
-          .and('have.text', 'Failed to fetch the dashboard details');
-      }
-    );
+    ).as('getMetricDefinitions');
+
+    cy.visitWithLogin('monitor/cloudpulse');
+
+    //  Wait for the services and dashboard API calls to complete before proceeding.
+    cy.wait(['@fetchServices', '@fetchDashboard']);
+
+    // Selecting a dashboard from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Dashboard')
+      .should('be.visible')
+      .type(`${dashboardName}{enter}`)
+      .should('be.visible');
+
+    //  Select a Database Engine from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Database Engine')
+      .should('be.visible')
+      .type(`${engine}{enter}`)
+      .should('be.visible');
+
+    //  Select a region from the dropdown.
+    ui.regionSelect.find().click().type(`${region}{enter}`);
+
+    // Select a resource (Database Clusters) from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Database Clusters')
+      .should('be.visible')
+      .type(`${clusterName}{enter}`)
+      .click();
+    cy.findByText(clusterName).should('be.visible');
```

Comment:
why we need this validations here, it is not needed as it is covered in a separate test file